extras/webdriver-installation/src/webdriver.rs

```diff
@@ -222,7 +222,7 @@ impl WebdriverKind {
                     .take()
                     .unwrap()
                     .channels
-                    .remove(&ChromeDriverReleaseDetailsChannel::Stable)
+                    .remove(&ChromeDriverReleaseDetailsChannel::Canary)
```

Comment:
What was the reason for this change?

Reply:
The web driver was crashing after switching to run a macOS runner, upgrading to the latest Chrome resolves the crash.

Reply:
That would have been a perfect commit message for a commit with just that change. ;-)

Reply:
True, I'll extract it

Reply:
I'll also see if I can use something less bleeding edge than Canary